### DISCUSSION TITLE
Svc-resample returns 0 on Paperspace.

### DISCUSSION THREAD

**Discussion by jonnytri53** [voicepaw/so-vits-svc-fork#625] [Q&A] [answered]
![Screenshot 2023-05-12 192816](https://github.com/voicepaw/so-vits-svc-fork/assets/132611653/21ae85a9-bbb8-4fc6-9e5e-24c3853f566b)
Hi, everyone.
I'm attempting to set up svc-vits on Paperspace, and everything appears to be working OK until I type the svc-resample command. I have 300 samples and have already checked the dataset_raw folder to ensure that the sample is present. However, when I use the svc-resample command. It's simply stuck at 0 samples. Thank you, everyone, for helping me.
P/s: I'm using P5000 GPU with 16GB VRAM

**Comment by Grownz** [comment]
I have the same issue on my local env as described here: https://github.com/voicepaw/so-vits-svc-fork/discussions/575

**Reply by Grownz** [reply, marked as answer]
Kind of answered my own issue. Maybe it'll do it for you?

**Reply by jonnytri53** [reply]
Thank you. It works perfectly.
